src/libs/API.js

```diff
@@ -105,7 +105,7 @@ function makeRequestWithSideEffects(command, apiCommandParameters = {}, onyxData
  * @param {Object} [onyxData.failureData] - Onyx instructions that will be passed to Onyx.update() when the response has jsonCode !== 200.
  */
 function read(command, apiCommandParameters, onyxData) {
-    makeRequestWithSideEffects(command, apiCommandParameters, onyxData, CONST.API_REQUEST_TYPE.READ);
+    SequentialQueue.getIsReadyPromise().then(() => makeRequestWithSideEffects(command, apiCommandParameters, onyxData, CONST.API_REQUEST_TYPE.READ));
```

Comment:
It would be great if we can document why we are waiting for the sequential queue here. If I was trying to understand how the API works as a new contributor or employee I'm not sure if this would be very obvious to me.